provider/storage_benchmark.py

```diff
@@ -46,7 +48,7 @@ class StorageBenchmark(object):
                         '_kill_pid': 'TASKKILL /F /IM %s /T',
                         '_rm_file': 'RD /S /Q "{}"'}}
     tar_map = {'.tar': '-xvf', '.tar.gz': '-xzf',
-               '.tar.bz2': '-xjf', '.tar.Z': '-xZf'}
+               '.tar.bz2': '-xzf', '.tar.Z': '-xZf'}
```

Comment:
Why replace  `j` with `z` ?   From my checking, `j` is correct. 
```
       -j, --bzip2
              Filter the archive through bzip2(1).

       -z, --gzip, --gunzip, --ungzip
              Filter the archive through gzip(1).
```

Reply:
thanks, have updated.